tox.ini

```diff
@@ -0,0 +1,18 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+# pip is broken on py32
+envlist = py23, py24, py25, py26, py27, py30, py31, py33, py34, py35, py36, py37
```

Comment:
Please remove py23, py24 and py25 as  they're not supported and it looks like Tox is not testing them correctly. I would expect them to fail (missing a json module).